server/src/test/java/org/cloudfoundry/identity/uaa/provider/saml/idp/ZoneAwareIdpMetadataGeneratorTest.java

```diff
@@ -20,20 +21,22 @@
 import org.springframework.security.saml.util.SAMLUtil;
 
 import java.security.Security;
+import java.util.Arrays;
 import java.util.List;
 
 import static org.cloudfoundry.identity.uaa.provider.saml.ZoneAwareMetadataGeneratorTests.cert1Plain;
 import static org.cloudfoundry.identity.uaa.provider.saml.ZoneAwareMetadataGeneratorTests.cert2Plain;
 import static org.cloudfoundry.identity.uaa.provider.saml.ZoneAwareMetadataGeneratorTests.samlKey1;
 import static org.cloudfoundry.identity.uaa.provider.saml.ZoneAwareMetadataGeneratorTests.samlKey2;
 import static org.cloudfoundry.identity.uaa.provider.saml.idp.SamlTestUtils.getCertificates;
-import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.*;
```

Comment:
Set IDE to not generate `*` imports. That way we know what classes are actually used.